packages/web/src/app/hooks/useCoinDrop.ts

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+export const useCoinDrop = () => {
```

Comment:
I originally moved this into its own hook thinking I'd reuse some of the things here like `getRootProps` but I didn't end up using it that way due to some bugs.

Decided to keep the hook here as it helped clean up the component a bit and simplified the claim endpoint (no need to re-upload the coin image).